prow/plugins.yaml

```diff
@@ -266,6 +266,16 @@ plugins:
   kubernetes/kubernetes-template-project:
   - approve
 
+  kubernetes/kube-openapi:
+  - approve
+  - assign
+  - help
+  - hold
+  - lgtm
+  - size
+  - trigger
+  - wip
```

Comment:
you're already getting most of these plugins from the org-level `kubernetes:`

what you probably want is:
```
- approve
- blunderbuss
```

And possibly `trigger` if you're using prow to run presubmit jobs

Reply:
agree. that's what i have in `cloud-provider-openstack` above